src/Translate.elm

```diff
@@ -182,6 +183,12 @@ translate language appString =
                     , it = str ++ " per bobina"
                     }
 
+                QuadraphonicStr ->
```

Comment:
Dear @retokromer, do you know if my translations for _quadraphonic (4-track)_ are idiomatic? "Quatre pistes" is how it's written in the TC-04, and for the Italian terminology I checked [wikipedia](https://it.wikipedia.org/wiki/Quadrifonia), but I'd love a second opinion.

Reply:
Is _quadraphonic_ in English a noon or an adjective? If an adjective, then it’s OK; but when a noon, then I would suggest to use a noon as well (FR: _quadriphonie_ and IT: _quadrifonia_ [_quadrafonia_ would also be possible, but I guess this sounds… older]). I personally prefer the noon solution, but it’s just my taste.

Reply:
Ah, I guess I do understand my hesitation: quadraphonic as noon or quadraphonic [sound] as adjective.

Reply:
Thanks, @retokromer!

Quadraphonic is an adjective, yes. Which fits with the rest of the audio configs, which are all descriptors for an audio reel.

PS French has _quadriphonie_, Italian _quadrifonia_, but English does not have the equivalent word _quadraphonia_, it's rather _quadraphonic sound_. There's The Who album _Quadrophenia_ though!